test/src/main/java/org/axonframework/test/aggregate/FixtureConfiguration.java

```diff
@@ -96,6 +96,15 @@
  */
 public interface FixtureConfiguration<T> {
 
+    /**
+     * Registers a subtype of this aggregate to support aggregate polymorphism. Command Handlers defined on this subtype
```

Comment:
I think this line should've referenced it's about several `subtypes` rather than a single one.